app/controllers/annotations_controller.rb

```diff
@@ -707,10 +707,14 @@ def parse_tags(resource)
       parsed_tags << tag
     end
     parsed_tags
-  end 
+  end
 
   def check_anno_auth(request, annotation)
-    AnnoAuthValidator.authorize(request.headers['Authorization'], getTargetingAnnosCanvas(annotation))
+    if Rails.application.config.use_jwt_auth
+      AnnoAuthValidator.authorize(request.headers['Authorization'], getTargetingAnnosCanvas(annotation))
+    else
+      true
+    end
```

Comment:
Sshould write a test to evaluate this functionality?

Reply:
Added the test.
Also deleted all webacl related lines because we don't use webacl anyways and they were a big hindrance to readability.